### ISSUE TITLE
Wiki doesn't tell where to download

### ISSUE THREAD

**Issue by ukdtom** [ukdtom/ClaimPMS4Win#2]
Please add

**Comment by trumpy81** [comment]
Download link for the latest release can be found on the home page and the how to page.

**Comment by ukdtom** [comment]
Do feel free to alter the Readme.md as well ;)

**Comment by trumpy81** [comment]
I am unable to edit the README.md, the edit button is greyed out after clicking on the file name and there is no edit pencil on the main <> code page.

I assume I do not have the necessary permissions to edit the file?

**Comment by ukdtom** [comment]
Huh.....

But you did edit it?

![image](https://user-images.githubusercontent.com/4699449/190102767-89fcfdc4-6d38-4dc1-a0cf-675d254567fd.png)

**Comment by trumpy81** [comment]
When I posted that I was unable to edit the file, but then I went back 3 hours or so later on and I was able to edit the file?

I thought you had updated permissions or something?

There are some commits waiting to be approved for README.md on the WTInstaller and ChuckPa's WTInstaller pages as well.

**Comment by ukdtom** [comment]
Didn't change anything, and you should also have the needed rights on wtinstaller already?

**Comment by trumpy81** [comment]
No, I don't have permission to edit the README.md file at all. If I click on the file name, the edit pencil icon is greyed out and there is no pencil icon on the main <> code page at all.

It's the same on Chuck's WTInstaller <> code page also.

**Comment by trumpy81** [comment]
I added a Downloads page to the Wiki ... just to be complete ... lol
